src/CMakeLists.txt

```diff
@@ -0,0 +1,323 @@
+cmake_minimum_required(VERSION 3.2)
+project(QZXing)
+
+find_package(Qt5 REQUIRED Core)
+find_package(Qt5 REQUIRED Network)
+find_package(Qt5 REQUIRED Xml)
+find_package(Qt5 REQUIRED Multimedia)
+find_package(Qt5 REQUIRED MultimediaWidgets Widgets)
+find_package(Qt5 REQUIRED Svg Quick QuickControls2)
+
+set(CMAKE_AUTOMOC ON)
+#set(CMAKE_BUILD_TYPE Debug)
+add_subdirectory(zxing)
+add_library(qzxing STATIC
+
+# root
+    CameraImageWrapper.cpp
+    CameraImageWrapper.h
+    ImageHandler.cpp
+    ImageHandler.h
+    QZXing.cpp
+    QZXing.h
+    QZXingFilter.cpp
+    QZXingFilter.h
+    QZXingImageProvider.cpp
+    QZXingImageProvider.h
+    QZXing_global.h
+
+# zxing/bigint
+
+    ${CMAKE_CURRENT_SOURCE_DIR}/zxing/bigint/BigInteger.hh
```

Comment:
do you really need `${CMAKE_CURRENT_SOURCE_DIR} ` here (and on all following lines?)

Reply:
I wrote this as a beginner, so all internal CMakeLists have been removed, so $ {CMAKE_CURRENT_SOURCE_DIR} is required.